docs/variants/clevo_nv41/flashing_external.md

```diff
@@ -84,8 +109,19 @@ securely.
 Once you have the backups saved, proceed to flashing the chip - replace
 `[path]` with the path to the binary you want to write - e.g. `build/coreboot.rom`
 
+
+If stock firmware or coreboot version v0.5.0 or older is installed
+
+```bash
+# flashrom -p ch341a_spi -c GD25B128B/GD25Q128B -w [path] --ifd -i bios
+```
+
+If coreboot version v0.5.0 or higher is already installed, only the
```

Comment:
It's external flashing, so i'ts not from OS. Isn't the one command with bios region enough for external flashing?

Reply:
That's true. Removed